File: src/main/java/io/github/oleksivio/telegram/bot/api/model/method/message/EditOtherMessageCaption.java
package io.github.oleksivio.telegram.bot.api.model.method.message;

import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker;
import io.github.oleksivio.telegram.bot.core.model.responses.BooleanResponse;
import io.github.oleksivio.telegram.bot.core.model.method.message.EditMessageCaption;
import io.github.oleksivio.telegram.bot.core.model.responses.CommonResponse;

public class EditOtherMessageCaption extends EditMessageCaption<Boolean> {

    public EditOtherMessageCaption(ActionNetworker actionNetworker) {
        super(actionNetworker);
    }

    @Override
    protected Class<? extends CommonResponse<Boolean>> getResultWrapperClass() {
        return BooleanResponse.class;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/method/message/EditOtherMessageCaption.kt
package io.github.oleksivio.telegram.bot.api.model.method.message

import io.github.oleksivio.telegram.bot.core.controller.network.ActionNetworker
import io.github.oleksivio.telegram.bot.core.model.BooleanResponse
import io.github.oleksivio.telegram.bot.core.model.method.message.EditMessageCaption

class EditOtherMessageCaption(actionNetworker: ActionNetworker) : EditMessageCaption<Boolean>(actionNetworker) {

    override val resultWrapperClass = BooleanResponse::class
}
